app/services/auth.py

```
@@ -339,7 +341,27 @@ def run_oauth() -> None:
+                            localhost_hosts = {
+                                "localhost",
+                                "127.0.0.1",
+                                "::1",
+                                "host.docker.internal",
+                                "gateway.docker.internal",
+                            }
+                            host_lower = settings.oauth_host.lower()
+                            is_local = (
+                                host_lower in localhost_hosts
+                                or host_lower.endswith(".localhost")
+                                or host_lower.endswith(".local")
```

Ruff (0.14.10)

Call `endswith` once with a `tuple`

Merge into a single `endswith` call

(PIE810)